inference-engine/tests/functional/inference_engine/transformations/mvn_fusion_test.cpp

```diff
@@ -30,7 +30,7 @@ TEST(TransformationTests, MVNFusionTestOutside) {
         auto sub2 = std::make_shared<ngraph::opset6::Subtract>(input, mean2);
         auto const_2 = ngraph::opset6::Constant::create(ngraph::element::f32, ngraph::Shape{}, { 2 });
         auto power_sqr = std::make_shared<ngraph::opset6::Power>(sub2, const_2);
-        auto mean3_axes = ngraph::opset6::Constant::create(ngraph::element::f32, ngraph::Shape{ 3 }, { 1, 2, 3 });
+        auto mean3_axes = ngraph::opset6::Constant::create(ngraph::element::i32, ngraph::Shape{ 3 }, { 1, 2, 3 });
```

Comment:
Why didi you change precisions?

Reply:
As we discussed in email thread, in these tests nGraph is built manually and for ReduceMean op, axes precision is incorrectly set as f32, while it does not have much sense as axes comprises indexes of input dimension. It is basically the same reason why LPT test precisions were changed.